### PR TITLE
typo in roiModeSupported  comment 

### PR DESCRIPTION
**Pull request by roiegeron-spearuav**
Fixed small typo in comment

### CODE REVIEW

File: src/FlightDisplay/FlyViewMap.qml [hunk 1]
```diff
@@ -533,13 +533,13 @@ FlightMap {
...
         onClicked: {
-            if (guidedActionsController.guidedUIVisible || (!guidedActionsController.showGotoLocation && !guidedActionsController.showOrbit)) {
+            if (guidedActionsController.guidedUIVisible || (!guidedActionsController.showGotoLocation && !guidedActionsController.showOrbit && !guidedActionsController.showROI)) {
                 return
             }
             orbitMapCircle.hide()
             gotoLocationItem.hide()
             var clickCoord = _root.toCoordinate(Qt.point(mouse.x, mouse.y), false /* clipToViewPort */)
-            if (guidedActionsController.showGotoLocation && guidedActionsController.showOrbit) {
+            if (guidedActionsController.showGotoLocation && (guidedActionsController.showOrbit || guidedActionsController.showROI)) {
```
Comment (DonLakeFlyer) [inline src/FlightDisplay/FlyViewMap.qml:542]: This isn't correct. It won't do the right thing if goto is not shown but orbit and roi are. This block should be change like this:

```
if (guidedActionsController.showGotoLocation && !guidedActionsController.showOrbit && !guidedActionsController.showROI) {
    ...
}  else if (guidedActionsController.showOrbit && ! guidedActionsController.showGotoLocation && !guidedActionsController.showROI) {
    ....
} else if (...) {
    ....
} else {
    Handle the case where a menu needs to be shown to pick from
}

Comment (roiegeron-spearuav) [reply src/FlightDisplay/FlyViewMap.qml:542]: of goto is enabler for ROI and Orbit then the if case is very simple
I made the changes and committed

Comment (DonLakeFlyer) [reply src/FlightDisplay/FlyViewMap.qml:542]: This is still incorrect since it doesn't follow what I showed above.

**Comment by DonLakeFlyer** [comment]
In looking through this I decided to simplify the menu handling to always show to menu even when only a single item. I also found other bugs in _hideROI and _hideOrbit and fixed them and replaced this pull with #8893. Thanks for finding the problem.

**Pull request closed without merging**